project/Tests/Snapper.Internals.Tests/Snapper.Internals.Tests.csproj

```diff
@@ -1,16 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net452</TargetFrameworks>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
why not test on 472 or 48?

Reply:
I've updated it to .net 7 in the latest changes. 

Since Snapper targets `.netstandard`, there isn't really any need to target .net framework specifically.

Reply:
ok, was just to be safe. 🙃

Reply:
I'm wanting to move snapper builds to be on linux fully in the future so unless there is a specific need for .net framework I wanted to avoid it 😅

Reply:
ok 😅